src/components/report/VariableControls.css

```diff
@@ -50,6 +72,12 @@
 }
 
 .instance-button {
+	padding: 0.25rem 0.5rem;
```

Comment:
There are quite a lot of magic spacing values here.
Could you extract these into some variables perhaps?

I really like to limit choices when it comes to padding, margin, border radius etc, to make sure you don't end up with 28 different spacings :-)

Reply:
The CSS is using steps of quarters of em/rem: 0.25 and 0.5, and in a rare occasion 0.75. I personally find this clear enough, and it prevents the 28 different values.

That said, some refactoring on the CSS side is perhaps in order. I'd like to do that in a separate PR if you don't mind! :-)

Reply:
Sure!